cea/demand/thermal_loads.py

```diff
@@ -101,7 +100,7 @@ def calc_thermal_loads(building_name, bpr, weather_data, usage_schedules, date,
     schedules = usage_schedules['schedules']
 
     # get n50 value
-    n50 = bpr.architecture['n50']
+    # n50 = bpr.architecture['n50']
```

Comment:
This is part of bpr. It can be called in every function that has bpr as an input.